lib/resources/auth_manager.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+
+/*
+  AuthManager class manages user authentication token.
+ */
+class AuthManager {
+
+  factory AuthManager() {
+    return _instance;
+  }
+
+  AuthManager._internal();
+
+  static final AuthManager _instance = AuthManager._internal();
+
+  static const _accessTokenKey = 'access_token';
+
+  void setAccessToken(accessToken) {
+    FlutterSecureStorage().write(key: _accessTokenKey, value: accessToken);
+  }
+
+  Future<String> getAccessToken() async {
+   return FlutterSecureStorage().read(key: _accessTokenKey);
```

Comment:
Would it make sense to store it in a variable in init probably instead of reading from disk storage every time.

Reply:
Yes. I can do that. I thought it is handled in SecureStorage itself and it is not.

Reply:
Added the logic to read it from variable and initializing the variable if it is null.